tgui/packages/tgui/interfaces/PreferencesMenu/data.ts

```diff
@@ -88,6 +88,7 @@ export type QuirkInfo = {
   max_positive_quirks: number;
   quirk_info: Record<string, Quirk>;
   quirk_blacklist: string[][];
+  species_quirk_blacklist: string[][];
```

Comment:
```suggestion
  species_quirk_blacklist: Record<string, string[]>;
```
Incorrect datatype here. `species_quirk_blacklist` is an associative list.